project/Settings.scala

```diff
@@ -0,0 +1,56 @@
+import Dependencies._
+import Merging._
+import Testing._
+import Version._
+import sbt.Keys._
+import sbt._
+import sbtassembly.AssemblyPlugin.autoImport._
+
+object Settings {
+
+  val artifactory = "https://artifactory.broadinstitute.org/artifactory/"
+
+  val commonResolvers = List(
+    "artifactory-releases" at artifactory + "libs-release",
+    "artifactory-snapshots" at artifactory + "libs-snapshot"
+  )
+
+  //coreDefaultSettings + defaultConfigs = the now deprecated defaultSettings
+  val commonBuildSettings = Defaults.coreDefaultSettings ++ Defaults.defaultConfigs ++ Seq(
+    javaOptions += "-Xmx2G",
```

Comment:
This refactor is based off rawls' template. I saw this setting in rawls, realized we didn't have it in orch, and thought it would be a good idea, so I kept it.